CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ aws-parallelcluster-node CHANGELOG
 
 This file is used to list changes made in each version of the aws-parallelcluster-node package.
 
+x.x.x
+------
+
+**ENHANCEMENTS**
+- Enable fast insufficient capacity fail-over with slurm scheduler.
```

Comment:
I would reword this to something like: "Temporarily disable compute resource when a node launch fails due to insufficient capacity"

Reply:
Reworded